components/tab/src/tab-bar/scroll-bar.js

```diff
@@ -1,27 +1,85 @@
 import propTypes from '@dhis2/prop-types'
-import { colors } from '@dhis2/ui-constants'
 import { IconChevronRight16, IconChevronLeft16 } from '@dhis2/ui-icons'
 import cx from 'classnames'
 import React, { Component, createRef } from 'react'
 import { animatedSideScroll } from './animated-side-scroll.js'
 import { detectHorizontalScrollbarHeight } from './detect-horizontal-scrollbar-height.js'
+import { ScrollButton } from './scroll-button.js'
 
 class ScrollBar extends Component {
     scrollBox = createRef()
     scrollArea = createRef()
-    state = {
-        scrolledToStart: false,
-        scrolledToEnd: false,
-    }
     horizontalScrollBarHeight = detectHorizontalScrollbarHeight()
+    disconnectResizeObserver = null
```

Comment:
I would probably prefer to create the resize observer in the constructor, start observing in `componentDidMount` and disconnect in `componentWillUnmount`.
_Note: this suggestion only makes sense if everyone is on board with making this the default behaviour._

Reply:
Done